docs/context/ref/Message.md

```diff
@@ -0,0 +1,78 @@
+---
+id: Message
+sidebar_label: Message
+title: Message
+hide_title: true
+---
+# `Message`
+
+A chat message to be sent through an instant messaging application. Can contain one or several text bodies (organised by mime-type, plaintext or markdown), 
+as well as attached entities (either arbitrary file attachments or fdc actions to be embedded in the message). To be put inside a ChatInitSettings object.
```

Comment:
Will have other uses (via SendChatMessage), ensure description is updated in that PR